client/components/App.js

```diff
@@ -2,16 +2,28 @@ import React, { useEffect } from "react"
 import { hot } from "react-hot-loader/root"
 import "foundation-sites"
 import $ from "jquery"
-import "../assets/scss/main.scss"
+import { Route, Switch, BrowserRouter, Redirect } from "react-router-dom"
 
-import { Route, Switch, BrowserRouter } from "react-router-dom"
+import "../assets/scss/main.scss"
+import PetTypesList from "./PetTypesList"
```

Comment:
As you start to add more components to this file, I might recommend creating a separation between external library imports and internal component imports - list out the external imports, add a new line, then list your internal component imports